pb_model/fields.py

```diff
@@ -103,6 +104,32 @@ def from_pb(instance, dj_field_name, pb_field, pb_value):
         raise NotImplementedError()
 
 
+class ProtoBufAnyField(models.BinaryField, ProtoBufFieldMixin):
+    @staticmethod
+    def to_pb(pb_obj, pb_field, dj_field_value):
+        getattr(pb_obj, pb_field.name).MergeFrom(dj_field_value)
```

Comment:
After checking `Any` API, I'm curious the reason of using `getattr(pb_obj, pb_field.name).MergeFrom(dj_field_value)` instead of `setattr(pb_obj, pb_field.name, Any().CopyFrom(dj_field_value)`. Will merge result into conflicts if there is already holding data unexpectedly?

Reply:
It was merely a mistake, i saw that `copyFrom` uses `mergeFrom` and confused myself, will change to `copyFrom`